samples/SpectatorView.Example.Unity/README.md

```diff
@@ -1,3 +1,70 @@
-# SpectatorView General Sample
+# SpectatorView.Example.Unity
 
-This sample exercises many aspects of SpectatorView with QR Code localization technique. For instructions on configuring, building and deploying please refer to the main [setup instructions](../../doc/SpectatorView.Setup.md).
\ No newline at end of file
+This sample shows how to use [QR Code detection](https://docs.microsoft.com/en-us/windows/mixed-reality/qr-code-tracking) to spatially align devices in the physical world.
+
+## Running the Demo
+
+In order to run the demo, you will need a HoloLens and an AR capable mobile device. Once you have the devices ready, follow the instructions below.
+
+> **ARKit Note:** If you wish to run the experience on an iOS device with ARKit, download the [ARKit repository](https://bitbucket.org/Unity-Technologies/unity-arkit-plugin/downloads/), unzip it, and copy the contents of the `Unity-Technologies-unity-arkit-plugin-94e47eae5954/Assets` folder to `/samples/Build2019Demo.Unity/Assets/`.
+
+### Instructions
+
+These instructions assume that a HoloLens device will be used to host the shared experience and that an Android phone will be used to spectate the shared experience. For iOS, replace references to Android with iOS.
+
+#### Prepare your local codebase
+
+1. Run `tools/Scripts/SetupRepository.bat` (`tools/Scripts/SetupRepository.sh` on Mac) to setup your local version of the code base. This script will obtain the external dependencies required for building this project.
+2. If you are using a HoloLens 1 device, build the `SpectatorView.OpenCV.dll` plugin and include it in the SpectatorView.Unity codebase. Instructions for building and adding `SpectatorView.OpenCV.dll` to the Spectator View codebase can be found [here](../../src/SpectatorView.Native/README.md).
+
+#### Prepare your devices
+
+1. Connect your devices to the same WiFi network.
+2. Obtain and write down your HoloLens's IP Address (This can be found in the settings menu).
+
+#### Configure your project
+
+1. Open the `SpectatorView.Example.Unity` project in Unity.
+2. If building for HoloLens 2, ensure that `QRCODESTRACKER_BINARY_AVAILABLE` has been defined as a Scripting Define Symbols in your WSA Unity Player Settings. If you have made no changes to the SpectatorView.Examples.Unity project definition, this should already be defined (Player settings can be found under `Build Settings -> Player Settings`).
+3. If building for HoloLens 1, ensure that `SpectatorView.OpenCV.dll` exists in the project's Assets folder. It should have been placed under Assets\MixedReality-SpectatorView\SpectatorView.Native\Plugins\WSA\x86 when going through the compilation instructions [here](../../src/SpectatorView.Native/README.md).
```

Comment:
If I follow the instructions here:

https://github.com/microsoft/MixedReality-SpectatorView/blob/master/src/SpectatorView.Native/README.md#4-adding-compiled-binaries-to-spectatorviewunity

And follow the entire page (which at a cursory glance seems like what I should do), does that script do the right thing, or does it copy too much? i.e. does the link that this goes to put the person on the right path (in which case are the specifics here just informational, instead of prescriptive)?

Usually when I see slightly differring levels of instructions I get confused as to which ones I should follow - this seems to say do one thing, the thing linked says more, is this a subset of the instructions, or is there some intersection here as well?

Reply:
removed unclear items

---
In reply to: [352945819](https://github.com/microsoft/MixedReality-SpectatorView/pull/290#discussion_r352945819) [](ancestors = 352945819)